docs/docs/environment_config.md

```diff
@@ -160,6 +160,16 @@ Any templated value can be supplied with a default value with the syntax:
 {% raw %}{{ var.value | default("default_value") }}{% endraw %}
 ```
 
+### String Manipulation
```

Comment:
I think this is expanding on Jinja templating functionality - I feel this section should merge with the section above (`Template Defaults`) to be a more general "Jinja Templating" section; with a brief sentence saying what it is, and links to the Jinja docs

Reply:
Ok will split this out